comp/forwarder/eventplatform/eventplatformimpl/types/types.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package types contains types related to the eventplatform
+package types
+
+import "github.com/DataDog/datadog-agent/pkg/logs/message"
+
+// An EventPlatformForwarder forwards Messages to a destination based on their event type
+type EventPlatformForwarder interface {
```

Comment:
Instead of creating this file, did you consider using `eventplatform.Component` and remove `EventPlatformForwarder`?

Reply:
Actually I misread your message, I replaced `EventPlatformForwarder` with `Forwarder`, using `Component` instead would require some more changes I think